front/lib/iam/session.ts

```diff
@@ -84,6 +84,7 @@ interface MakeGetServerSidePropsRequirementsWrapperOptions<
   enableLogging?: boolean;
   requireUserPrivilege: R;
   requireCanUseProduct?: boolean;
+  requireUserInCurrentWorkspace?: boolean;
```

Comment:
`nit`: what about the other away around? like `authFromOtherWorkspace`?

Reply:
Done already hehe

Reply:
Yes felt weird when reviewing the code.